bib/cmd/bootc-image-builder/image.go

```diff
@@ -322,6 +322,26 @@ func manifestForDiskImage(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest
 	return &mf, err
 }
 
+func labelForISO(os *source.OSRelease, arch *arch.Arch) string {
+	var label string
+	switch os.ID {
+	case "fedora":
+		label = fmt.Sprintf("Fedora-S-dvd-%s-%s", arch, os.VersionID)
```

Comment:
(niptick) this could just be a `return fmt.Sprintf("Fedora-S-dvd-%s-%s", arch, os.VersionID)` and he samefor all the other cases